priv/graphql/schemas/user/inbox.gql

```diff
@@ -1,10 +1,10 @@
 """
 Allow user to flush own inbox bin".
 """
-type InboxUserMutation @protected{
+type InboxUserMutation @use(modules: ["mod_inbox"]) @protected{
   "Flush the user's bin and return the number of deleted rows"
   flushBin(
     "Remove older than given days or all if null"
     days: PosInt
-  ): Int 
+  ): Int @use
```

Comment:
We can try to make annotating fields unnecessary if the host type can be taken from an authorized user. But it could make the solution more complex when we still need to choose the arg with domain in the admin API.